packages/api/src/utils/server/registerRoute.ts

```diff
@@ -3,13 +3,15 @@ import {ServerInstance, RouteConfig, ServerRoute} from "./types.js";
 export function registerRoute(
   server: ServerInstance,
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  route: ServerRoute<any>
+  route: ServerRoute<any>,
+  namespace?: string
 ): void {
   server.route({
     url: route.url,
     method: route.method,
     handler: route.handler,
-    schema: route.schema,
+    // append the namespace as a tag for downstream consumption of our API schema, eg: for swagger UI
+    schema: {...route.schema, ...(namespace ? {tags: [namespace]} : undefined)},
```

Comment:
This was required in order to group paths by namespace.

It would be nice to be able to reuse more of the beacon API schema but :shrug: that seems like a larger refactor.